src/com/aopphp/go/util/PluginUtil.java

```diff
@@ -24,25 +28,20 @@
 
         // search for use alias in local file
         final Map<String, String> useImports = new HashMap<String, String>();
-        psiFile.acceptChildren(new PsiRecursiveElementWalkingVisitor() {
-            @Override
-            public void visitElement(PsiElement element) {
-                if(element instanceof PhpUse) {
-                    visitUse((PhpUse) element);
-                }
-                super.visitElement(element);
-            }
 
-            private void visitUse(PhpUse phpUse) {
-                String alias = phpUse.getAliasName();
-                if(alias != null) {
-                    useImports.put(alias, phpUse.getFQN());
-                } else {
-                    useImports.put(phpUse.getName(), phpUse.getFQN());
+        if (psiFile instanceof PhpFile) {
+            final List<PhpUseList> useLists = PhpCodeInsightUtil.collectImports((PhpPsiElement) psiFile);
```

Comment:
`PhpCodeInsightUtil.collectImports((PhpPsiElement) psiFile);` always returns empty list for me. 🤔